docs/api.md

```diff
@@ -429,10 +432,11 @@ puppeteer.launch().then(async browser => {
   await page.exposeFunction('readfile', async filePath => {
     return new Promise((resolve, reject) => {
       fs.readFile(filePath, 'utf8', (err, text) => {
-        if (err)
+        if (err) {
```

Comment:
This ins't the code style for the project. When if blocks have one statement executions the curlies are ignored.